gravitee-apim-console-webui/src/portal/customization/theme/portal-theme.component.html

```diff
@@ -16,21 +16,116 @@
-          <mat-error *ngIf="developerPortalThemeForm.controls.name.hasError('required')">Name is required.</mat-error>
-        </mat-form-field>
-      </form>
-    }
-  </mat-card-content>
-</mat-card>
-<gio-save-bar [form]="developerPortalThemeForm" (resetClicked)="reset()" (submitted)="submit()"></gio-save-bar>
+<div class="portal-theme">
+  <div class="portal-theme__left">
+    <div class="portal-theme__left__actions">
+      <button mat-raised-button color="primary" type="submit" (click)="submit()" [disabled]="this.formUnchanged">Publish</button>
+      <button mat-raised-button color="basic" type="button" (click)="reset()" [disabled]="this.formUnchanged">Discard Changes</button>
+    </div>
+    <mat-card class="portal-theme__left__form">
+      <div class="portal-theme__left__form__header">
+        <span class="gio-badge-accent"><mat-icon svgIcon="gio:chat-lines"></mat-icon>&nbsp;Applies to new portal only</span>
+        <button mat-stroked-button type="button" (click)="restoreDefaultValues()" matTooltip="Restore default values">
+          <mat-icon svgIcon="gio:refresh-ccw"></mat-icon>
+        </button>
+      </div>
+      <mat-card-content class="portal-theme__left__form__content">
+        <form *ngIf="portalThemeForm" [formGroup]="portalThemeForm">
+          <mat-accordion multi>
+            <mat-expansion-panel class="portal-theme__left__form__content__panel mat-elevation-z0" expanded>
+              <mat-expansion-panel-header>
+                <mat-panel-title>Logos</mat-panel-title>
+              </mat-expansion-panel-header>
+              <div class="portal-theme__left__form__content__panel__pictures">
+                <div class="portal-theme__left__form__content__panel__pictures__picture">
+                  <p>Logo</p>
+                  <gio-form-file-picker formControlName="logo" accept=".jpg, .jpeg, .png">
+                    <gio-form-file-picker-label>PNG, JPG up to 5MB</gio-form-file-picker-label>
+                  </gio-form-file-picker>
+                </div>
+                <div class="portal-theme__left__form__content__panel__pictures__picture">
+                  <p>Favicon</p>
+                  <gio-form-file-picker formControlName="favicon" accept=".jpg, .jpeg, .png">
+                    <gio-form-file-picker-label>PNG, JPG up to 5MB</gio-form-file-picker-label>
+                  </gio-form-file-picker>
+                </div>
+              </div>
+            </mat-expansion-panel>
+            <mat-expansion-panel class="portal-theme__left__form__content__panel mat-elevation-z0" expanded>
+              <mat-expansion-panel-header>
+                <mat-panel-title>Font</mat-panel-title>
+              </mat-expansion-panel-header>
+              <div class="portal-theme__left__form__content__panel__fonts">
+                <mat-form-field>
+                  <mat-label>Font</mat-label>
+                  <input matInput formControlName="font" />
```

Comment:
This should probably be a mat-select --> user can choose a font within a list of available fonts.

Reply:
However, if we don't want the user to be able to change the font for now, we can disable this form field.